letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
+    case integer
+        when (0..59)
+             'F'
+        when (60..69)
+             'D'
+        when (70..79)
+             'C'
+        when (80..89)
+             'B'
+        else
+            'A'
+        end
+ 
```

Comment:
It's a tiny thing, but I think this would be more readable if you used when... then to turn these into one-liner cases. Otherwise looks good to me